src/tike/operators/cupy/multislice.py

```diff
@@ -37,7 +40,11 @@ def __init__(
             **kwargs,
         )
         self.propagation = propagation(
-            detector_shape=detector_shape,
+            norm = norm,
+            probe_shape=probe_shape,
+            wavelength=probe_wavelength,
+            probe_FOV=probe_FOV_lengths,
+            distance=multislice_propagation_distance,
             **kwargs,
         )
```

Comment:
You probably don't need to explicitly pass all of these variables down the stack because keyword arguments are implicitly passed to the component classes using `**kwargs`.